python/paddle/fluid/tests/unittests/mkldnn/test_transpose_int8_mkldnn_op.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Shouldn't there be 2019?

Reply:
@luotao1 Shall we use 2019 or keep 2018? You probably have a tool to unify all the copyright year?

Reply:
> You probably have a tool to unify all the copyright year?

We don't unify all the copyright year, since it records the first created time. Thus, if a file is first created in 2019, it should be 2019.

Reply:
> > You probably have a tool to unify all the copyright year?
> 
> We don't unify all the copyright year, since it records the first created time. Thus, if a file is first created in 2019, it should be 2019.

Understood. Thanks.

Reply:
Thanks for the reminder, I have changed the license year in latest version.